docs/conops.md

```diff
@@ -0,0 +1,582 @@
+
+This document proposes an open software architecture to support dynamic airspace management, data exchange, and regulatory compliance between Urban Aerial Mobility (UAM) actors. The proposed system undertakes the responsibilities of **Provider of Services to UAM (PSU)** and **Provider of Supplemental Data and Services (SDSP)** as outlined by the [NASA UAM Airspace Research Roadmap](https://ntrs.nasa.gov/citations/20210019876). It also proposes use cases in the form of rideshare and cargo services.
+
+The proposed open source ecosystem attempts to capture the future of advanced aerial mobility, as it could be. It enables operators safe and secure access to airspace, manages emergency contingencies, complies with the data exchange systems of multiple regulatory bodies, and allows interoperability with other service providers. In a supplemental capacity, it offers automated maintenance scheduling, certification monitoring, telemetry rebroadcasts, and secure storage. Underscoring all systems is the concept of interoperability: that aircraft from numerous manufacturers, countries, and service providers may cooperate on the same network.
+
+### 1.0.2 Assumptions and Constraints
+This architecture acknowledges the existence of other Providers of Service (PSUs), that vertiports and aircraft will be owned and operated by various companies and individuals, and that compliance with civil aviation authority differs from region to region.
+
+This architecture intends to abide by standardized protocols established by NASA, EASA, and other agencies to ensure interoperability between UAM actors and systems.
+
+## 1.1 Overview of the Envisioned System
+
+### 1.1.1 Overview
+
+The Arrow Software Services are broken into several domains of responsibility.
+
+<center> 
+
+```mermaid
+graph TB
+classDef subgraph_padding fill:none,stroke:none
+
+subgraph Provider of Services
+    subgraph p [ ]
+        sched[svc-scheduler]
+        dcvr[svc-discovery]
+        com[svc-compliance]
+        guid[svc-guidance]
+    end
+end
+
+class p subgraph_padding
+```
+```mermaid
+graph TB
+classDef subgraph_padding fill:none,stroke:none
+
+subgraph Supplemental Data and Service Provider
+    subgraph p [ ]
+
+	stor[svc-storage]
+	tlm[svc-telemetry]
+	upk[svc-upkeep]
+
+    end
+end
+
+class p subgraph_padding
+```
+
+```mermaid
+
+graph TB
+subgraph Arrow Rideshare & Cargo Services
+classDef subgraph_padding fill:none,stroke:none
+
+    subgraph p [ ]
+	share[svc-rideshare]
+	cargo[svc-cargo]
+    charter[svc-charter]
+	pay[svc-payment]
+	contact[svc-contact]
+    end
```

Comment:
We also have svc-pricing now (-;

Reply:
https://github.com/Arrow-air/se-services/pull/2/commits/fd87e1512c5648b706fd9161d66063f763adc043